runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesConfig.java

```diff
@@ -98,24 +98,24 @@ default boolean warnForMagic()
 	}
 
 	@ConfigItem(
-		keyName = "hideAutoRetaliate",
-		name = "Hide auto retaliate",
-		description = "Hide auto retaliate from the combat options tab",
+		keyName = "autoRetaliateHandler",
+		name = "How to handle auto retaliate",
+		description = "Choose how auto retaliate button will be handled",
 		position = 7
 	)
-	default boolean hideAutoRetaliate()
+	default AutoRetaliateHandleType autoRetaliateHandler()
```

Comment:
Maybe you should change this to boolean Warn for autoretalliate, and then adding dropdown for either locking or hiding warned stuff or do nothing